### ISSUE TITLE
some region return nan values some are not 

### ISSUE THREAD

**Issue by ramdhan1989** [AlexeyAB/darknet#4758]
I got this result :
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 130 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: 0.504471, .5R: -nan, .75R: -nan, count: 0, loss = 74375.835938, class_loss = 74375.835938, iou_loss = 0.000000
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 82 Avg (IOU: 0.310754, GIOU: 0.223625), Class: 0.568450, Obj: 0.503603, No Obj: 0.473598, .5R: 0.222222, .75R: 0.111111, count: 9, loss = 276.509735, class_loss = 265.729645, iou_loss = 10.780090
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 94 Avg (IOU: 0.291759, GIOU: 0.105928), Class: 0.524678, Obj: 0.562712, No Obj: 0.553509, .5R: 0.333333, .75R: 0.000000, count: 9, loss = 1421.265015, class_loss = 1405.048584, iou_loss = 16.216431
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 106 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: 0.500648, .5R: -nan, .75R: -nan, count: 0, loss = 4664.142090, class_loss = 4664.142090, iou_loss = 0.000000
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 118 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: 0.473961, .5R: -nan, .75R: -nan, count: 0, loss = 16648.542969, class_loss = 16648.542969, iou_loss = 0.000000

is it normal or not ?

**Comment by lsd1994** [comment]
Normal, because some regions have no object.
